src/main/java/com/gtnewhorizons/obama/main/tileentities/multi/definition/structure/IConstructableStructure.java

```diff
@@ -212,4 +211,17 @@ default void registerStructure() {
      * @param hintsOnly the hints only mode switch
      */
     void construct(ItemStack itemStack, boolean hintsOnly);
+
+    /**
+     * <p>For hatch list clear, reset heat coil status or etc.
```

Comment:
Is the html p here javadoc or something?

Reply:
javadoc supports html and the left part of javadoc in it also uses <p>. so i directly use it